aten/src/ATen/core/type.cpp

```diff
@@ -1071,6 +1071,17 @@ void ClassType::addMethod(torch::jit::Function* method) {
   methods_.push_back(method);
 }
 
+void ClassType::addOverloadedMethod(torch::jit::Function* method) {
+  auto it = overloaded_methods_.insert(
```

Comment:
similar here, guard against the case where `method_name` already exists as key in map.

Reply:
yep.

Reply:
It checks now.